src/ProjectSystemDogfoodSetup/ProjectSystemDogfoodSetup.csproj

```diff
@@ -17,6 +17,7 @@
     <IncludeDebugSymbolsInVSIXContainer>false</IncludeDebugSymbolsInVSIXContainer>
     <IncludeDebugSymbolsInLocalVSIXDeployment>false</IncludeDebugSymbolsInLocalVSIXDeployment>
     <ImportVSSDKTargets>true</ImportVSSDKTargets>
+    <DeployExtension Condition="'$(DeployVsixExtension)' == 'false'">false</DeployExtension>
     <RestorePackages>true</RestorePackages>
```

Comment:
Why don't you just set DeployExtension when you call MSBuild so you don't have to do this and it just automatically works?